lib/presentation/shared/tile.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+
+class CustomTile extends StatelessWidget {
+  final String title;
+  final String subtitle;
+  final VoidCallback onTap;
+
+  const CustomTile({
+    Key? key,
+    required this.title,
+    required this.subtitle,
+    required this.onTap,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: MediaQuery.of(context).size.width * 0.95,
```

Comment:
why this sizeBox?

Reply:
keep the tile expanded, and just use horizontal padding.

Reply:
Okay
Remove splash and highlight color or not?

https://user-images.githubusercontent.com/50518325/218433404-30f22b6e-08f6-49d2-8211-72f2c054b7ba.mp4

Reply:
keep